### ISSUE TITLE
Change name of custom_functions.h 

### ISSUE THREAD

**Issue by audiologic-vannoy** [fpga-open-speech-tools/component_library#6]
The file is really just fixed point to string and vice versa conversion funcitons. There is probably a name that better describes what the file is actually doing. 

Changing the file name will necessitate changing the device driver autogen to reflect the new file name.

**Comment by fe-wickham** [comment]
Is there a reason we have the strcat2 function instead of just using strcat?

**Comment by tvannoy** [comment]
Unfortunately, yes. Kernel source code can't use the std library functions since they are written for user space. https://kernelnewbies.org/FAQ/LibraryFunctionsInKernel

However, it's entirely possible that there is already a kernel version of strcat available for use in the kernel source tree. If that's the case, it would be better to use that version instead of our (probably buggy) version.

**Comment by fe-wickham** [comment]
Given it is in [string.h](https://github.com/fpga-open-speech-tools/linux-socfpga/blob/openspeech/include/linux/string.h) though shouldn't we be able to use it that way?

**Comment by tvannoy** [comment]
Yes, you are correct. Whoever implemented the driver did not know that there is a kernel version of string.h that implements many of the functions in the user space string.h (located at /usr/include/string.h)

Let's use include/linux/string.h

**Comment by tvannoy** [comment]
There are probably other functions, such as strcpy, we could replace that I believe we are duplicating in our drivers.
